packages/E2ETest/metro.config.js

```diff
@@ -32,6 +32,19 @@ module.exports = {
       ),
     ]),
   },
+
+  // Metro doesn't currently handle assets from other packages within a monorepo.  This is the current workaround people use
+  server: {
```

Comment:
just for consideration: Since these are so identical, should we place it in a seperate js file and have something like `server: require('../shared.metro.config.js').server`